packages/terra-functional-testing/src/services/wdio-visual-regression-service/modules/makeElementScreenshot.js

```diff
@@ -28,6 +28,11 @@ async function makeElementScreenshot(browser, elementSelector, options = {}) {
 
   // get bounding rect of elements
   const boundingRects = await browser.execute(getBoundingRects, elementSelector);
+
+  if (boundingRects.length === 0) {
+    throw new Error(`[wdio-visual-regression-service:makeDocumentScreenshot] Failed to capture the element using the "${elementSelector}" selector. Either update the test document to include this selector or use a different selector that exists on the document.`);
```

Comment:
Do we want to use the [terra-cli logger](https://engineering.cerner.com/terra-ui/dev_tools/cerner/terra-cli/logger) here? I'm not sure what our rule of thumb is between throwing errors and using the logger to output the error. Throwing an error might make more sense here to break out of the function

Reply:
We want to fail the test in this situation. Using terra-cli logger to log an error does not fail the test and we would still need to return a value (string) which would need to be handled downstream.